manifests/s/SublimeHQPtyLtd/SublimeText/4/Dev/4.0.0.413800/SublimeHQPtyLtd.SublimeText.4.Dev.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: SublimeHQPtyLtd.SublimeText.4.Dev
+PackageVersion: 4.0.0.413800
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://download.sublimetext.com/sublime_text_build_4138_x64_setup.exe

```

Comment:
It looks like these releases are not specific dev builds, meaning that when this is promoted to be a stable build, the dev package will have conflicts with the stable package `SublimeHQ.SublimeText.4` so each time the build moves from dev to stable the manifest will have to be moved.